### ISSUE TITLE
Stuck on loading in v0.32a jar

### ISSUE THREAD

**Issue by iakshay** [uwplse/dviz#24]
It seems like the app.js is not linked with `react-json-tree` module in the 0.32 release jar. This is the error I get when I load the localhost:3000.

```
Uncaught TypeError: Cannot read property 'react-json-tree' of undefined at app.js:1453
```

**Comment by iakshay** [comment]
Also, I just tried `v0.31a` release and that works fine.

**Comment by dwoos** [comment]
I tried to fix this in `v0.33a`, but it's still not working. I'll keep looking at it--for now, you can use the `v0.31a` release. Thanks for finding this issue!

**Comment by dwoos** [comment]
This is fixed in `v0.35a`. Thanks again for finding it!
